packages/medusa-payment-stripe/src/api/routes/hooks/stripe.js

```diff
@@ -90,3 +103,23 @@ export default async (req, res) => {
     await handleCartPayments(event, req, res, cartId ?? resourceId)
   }
 }
+
```

Comment:
question: My IDE complains that `paymentIntentId` is undefined on line 101. We should use `paymentIntent.id`. Do you get the same?

Reply:
I don't get anything, let me see

Reply:
ok it was not shown as an error cause it is js i ve fixed it